isaric/parsers/brazil-followup.toml

```diff
@@ -0,0 +1,829 @@
+    { field = "flw_ethnicity_oth" },
+  ]
+
+[subject.pregnancy]
+  field = "flw_prg"
+  ref = "Y/N"
+  if = { flw_sex_at_birth_new = 2 }
+
+[subject.pregnancy_gestational_age_weeks]
+  field = "flw_preg_gest"
+
+[subject.has_chronic_hematologic_disease]
+  field = "flw_spc_ongoing___12"
+  ref = "checkbox"
+
+[subject.has_asplenia]
+  field = "flw_spc_ongoing___10"
+  ref = "checkbox"
+
+[subject.has_tuberculosis]
+  field = "flw_spc_ongoing___17"
+  ref = "checkbox"
+
+[subject.has_dementia]
+  field = "flw_spc_ongoing___15"
+  ref = "checkbox"
+
+[subject.has_chronic_respiratory_disease]
+  field = "flw_spc_ongoing___4"
+  ref = "checkbox"
+
+[subject.has_rheumatologic_disorder]
+  field = "flw_spc_ongoing___13"
+  ref = "checkbox"
+
+[subject.has_hiv]
+  field = "flw_spc_ongoing___16"
+  ref = "checkbox"
+
+[subject.has_hypertension]
+  field = "flw_spc_ongoing___2"
+  ref = "checkbox"
+
+[subject.has_malignant_neoplasm]
+  field = "flw_spc_ongoing___11"
+  ref = "checkbox"
+
+[subject.has_smoking]
+  field = "flw_smoking"
+
+  [subject.has_smoking.values]
+    1 = "current" # "I do this more often"
+    # 2 = "yes" # "I do it less often"
+    3 = "current" # "no difference"
+    4 = "no"      # "I Didn't Do This Before COVID-19"
+
+[subject.has_asthma]
+  field = "flw_spc_ongoing___3"
+  ref = "checkbox"
+
+[subject.has_chronic_cardiac_disease]
+  field = "flw_spc_ongoing___1"
+  ref = "checkbox"
+
+[subject.has_chronic_kidney_disease]
+  field = "flw_spc_ongoing___8"
+  ref = "checkbox"
+
+[subject.has_diabetes]
+  combinedType = "any"
+  fields = [
+    { field = "flw_spc_ongoing___5", ref = "checkbox" },
+    { field = "flw_spc_ongoing___6", ref = "checkbox" },
+    { field = "flw_spc_ongoing___7", ref = "checkbox" },
+  ]
+
+[subject.diabetes_type]
+  combinedType = "firstNonNull"
+  fields = [
+    { field = "flw_spc_ongoing___5", values = { 1 = "type-1" } },
+    { field = "flw_spc_ongoing___6", values = { 1 = "type-2" } },
+    { field = "flw_spc_ongoing___7", values = { 1 = "gestational" } },
+  ]
+
+[subject.has_liver_disease]
+  field = "flw_spc_ongoing___9"
+  ref = "checkbox"
+
+[subject.has_comorbidity_other]
+  combinedType = "set"
+  excludeWhen = "none"
+  fields = [{ field = "flw_chron_other" }]
+
+[subject.icu_admitted]
+  field = "flw_hosp_icu_new"
+  description = "Admission to an Intensive Care Unit (ICU / CTI)"
+  ref = "Y/N"
+
+  # VISIT
+  # required fields - "visit_id", "subject_id", "country_iso3", "start_date", "outcome", "date_outcome"
+
+[visit]
+  country_iso3 = "BRA"
+
+  [visit.visit_id]
+    field = "flw_subjid"
+    sensitive = true
+
+  [visit.subject_id]
+    field = "flw_subjid"
+    sensitive = true
+
+  [visit.start_date]
+    field = "flw_first_admit"
+
+  [visit.pathogen_test_date] # is this correct?
+    combinedType = "firstNonNull"
+    fields = [
+      { field = "flw_date_test", if = { flw_c19_testresult = 1 } },
+      { field = "flw_date2_test" },
```

Comment:
this condition is fine for flw_date_test if we are only capturing positive test results.